### ISSUE TITLE
How can I hook into onPause/onEnded?

### ISSUE THREAD

**Issue by laduke** [agrasley/redux-audio#1]
Hi, thanks for making this. 
I have a list of audio files, and I want to automatically go to the next one when one ends. So I need know when playing ends. The only thing I can get to work is through store.subscribe(). 

It seems like I should be able to do like `<Audio onPause={() => myPauseDispatcher() } ... />` 

Is there a way to do it?

**Comment by agrasley** [comment]
My recommendation is to use [redux-saga](https://github.com/yelouafi/redux-saga) to schedule sequential actions like you're suggesting. Otherwise, you can definitely make a pull request with the ability to hook into the functions.

**Comment by laduke** [comment]
I got a couple things working. Wondering what you think. Basicially I need onEnded and friends exposed on Audio's render(). 

```
    return (
      <audio
        autoPlay={autoPlay}
        controls={controls}
        loop={loop}
        preload={preload}
        src={src}
        onPause={onPause}
        onEnded={onEnded}
        onPlaying={onPlaying}
      />
    )
```

but the onEnded dispatch/prop name is already used by the library, so the user can't pass it in. 

So, we could: 
- Make user use a different property name ( `onEnded={onEnded2}` ) :/
- rename these in the library's mapDispatchToProps to like onEndedHandler so onEnded is available to the user. 
- "namespace" them using mergeProps into like reduxAudio.onEnded. 

I don't think any of this would break anything for existing users. (?)

I put what I started here https://github.com/laduke/redux-audio/commit/fd8efd2089e2b269a3cf4e906aa6e7b148940e83 but I'd still have to fix the propTypes.
